app/controllers/ObjectMatrixEntryMixin.scala

```diff
@@ -16,9 +19,9 @@ trait ObjectMatrixEntryMixin {
     val contentRangeHeader = ranges.headOption.map(range=>s"bytes ${range.headerString}${totalSize.map(s=>s"/$s").getOrElse("")}")
 
     val optionalFields = Seq(
-      entry.attributes.flatMap(_.stringValues.get("MXFS_MODIFICATION_TIME")).map(s=>"Etag"->s),
+      //entry.attributes.flatMap(_.stringValues.get("MXFS_MODIFICATION_TIME")).map(s=>"ETag"->s),
```

Comment:
this can be removed (etag is now set to appliance checksum in BulkDownloadController)